build/rubygem-simp-cli.spec

```diff
@@ -124,6 +124,9 @@ EOM
 %doc %{gemdir}/doc
 
 %changelog
+* Thu Jun 17 2021 Jeanne Greulich <jeanne.greulich@onyxpoint.com> - 7.0.0
+- The LOCAL sssd domain is no longer needed for sssd to start.  This
+  fix removes adding  LOCAL to list of domains.
```

Comment:
Please note this is a change to `simp config` and when it applies.  Also, please add a line between this and the previous changelog entry.